.rubocop.yml

```diff
@@ -0,0 +1,14 @@
+MethodLength:
+  Enabled: false
+ClassLength:
+  Enabled: false
+ModuleLength:
+  Enabled: false
+Metrics/AbcSize:
+  Enabled: false
+Documentation:
+  Enabled: false
+Style/ClassAndModuleChildren:
+  Enabled: false
+Metrics/LineLength:
+  Max: 100
```

Comment:
Minor: I like to comply with git preferences of having a new-line also in the last line (patches/diffs are cleaner).